noir-projects/noir-contracts/contracts/token_with_refunds_contract/src/test/basic.nr

```diff
@@ -1,100 +1,70 @@
-use crate::test::utils;
+use crate::{test::utils, TokenWithRefunds, types::token_note::TokenNote};
+
 use dep::aztec::{
     test::helpers::cheatcodes, oracle::unsafe_rand::unsafe_rand, hash::compute_secret_hash,
-    prelude::NoteHeader
+    prelude::NoteHeader, protocol_types::storage::map::derive_storage_slot_in_map
 };
-use crate::TokenWithRefunds;
-use crate::types::token_note::TokenNote;
 use dep::authwit::cheatcodes as authwit_cheatcodes;
 
-#[test]
-unconstrained fn transfer_success() {
```

Comment:
Nuked this test because it's no longer necessary as it's just a standard token transfer now.